bin/trigger_azure_pipelines.py

```diff
@@ -68,6 +71,35 @@ def trigger_azure_pipeline_and_wait_until_its_completed(
     if completed_run.result in FAILED_STATUSES:
         raise Exception("Azure DevOps pipeline run failed!")
 
+    return run_id
+
+
+def download_callback(chunk, response) -> None:
+    print(f"Downloaded chunk of size: {str(len(chunk))}")
+
+
+def download_artifact(
+    ado_client: Connection,
+    project: str,
+    run_id: str,
+    ado_artifact_name: str,
+    artifact_download_path: str,

```

Comment:
The method name contains the context that you are downloading an "artifact" so this arg name could be shortened.
```suggestion
    download_path: str,

```